### DISCUSSION TITLE
Set Checkbox to checked/unchecked in CheckboxList?

### DISCUSSION THREAD

**Discussion by RPotkowaVegasystems** [filamentphp/filament#8284] [Help]
I have a Checkboxlist with a relation to permissions. I would like to set with an Action to check/uncheck all checkboxes.

My Tagcode looks like

```
wire:model.live="mountedTableActionsData.0.permission" value="75...76... etc"
```

I tried:

```
$set('permission.76', true);
$set('permission.76', 'checked');
$set('permission, 76);
```

But nothing works.

**Comment by leandrocfe** [comment]
`->bulkToggleable()` ?

**Comment by RPotkowaVegasystems** [comment]
OK, thanks, This works. 
But when i would like it to make with my method, for example to make a specific action like  Select only 1,3,5th checkox inside multiple checkbox list etc., what is the right method to $set() the values?

**Reply by leandrocfe** [reply]
`$set('permission', [75, 76])` ?
Not sure where are you using this, but you can also try
`->action(fn ($livewire) => $livewire->data['permission'] = [75, 76])`

**Comment by RPotkowaVegasystems** [comment]
OK thanks! I could access on it with:
```
$livewire->mountedTableActionsData[0]['permission']
```
